projects/packages/videopress/src/client/block-editor/hooks/use-video-data-update/index.ts

```diff
@@ -306,6 +307,13 @@ export function useSyncMedia(
 				// Update local state with fresh video data.
 				updateInitialState( dataToUpdate );
 
+				// Privacy settings attribute update
+				if ( dataToUpdate?.privacy_setting !== 2 ) {
```

Comment:
Maybe we should use constants instead of numbers directly.

Reply:
yes, I think. it's idea because I always end up don't knowing what, in this case, `2` means 🤦

Reply:
follow-up?

Reply:
As you think is best.